lib/grantinee/configuration.rb

```diff
@@ -41,6 +41,8 @@ def url=(url)
         @engine = :postgres
       end
 
+      raise 'Invalid database url' unless uri.user && uri.host && uri.path
```

Comment:
(wondering) shouldn't it then have also port and database checked?

Reply:
port has a default and database is checked via `uri.path` 🙂